akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/BehaviorTestKitImpl.scala

```diff
@@ -201,8 +201,8 @@ private[akka] object BehaviorTestKitImpl {
     def inteceptBehaviour[T](behavior: Behavior[T], ctx: TypedActorContext[T]): Behavior[T] =
       Behavior
         .start(Behaviors.intercept { () =>
-          this.asInstanceOf[BehaviorInterceptor[Any, T]]
-        }(behavior), ctx.asInstanceOf[TypedActorContext[Any]])
+            this.asInstanceOf[BehaviorInterceptor[Any, T]]
```

Comment:
can we keep this two spaces?

Reply:
My impression is that these binpack options stated in https://github.com/mdedetrich/akka-apache/pull/2#discussion_r999500757 override what is meant to be correct and legitimate behaviour that was fixed and it appears that its causing conflicts with indentation.

As I noted right at the start of the PR, this is getting into the territory of bikeshedding and now that I think about it more, I am not so comfortable using such options because in the long run they cause more problems then they solve.  If my previous statement is correct that means we have to decide between consistent indentation and fixing a couple of exceptions.

Reply:
So I decided to revert the change, unless someone knows how to still keep the indentation consistent.

Reply:
in binpacking, one indentation level is added for every opening parenthesis, so what you saw is correct. however, some people prefer a solid mass of code without extra indentation, and there's a parameter for that. you should be able to find it easily.

Reply:
Are you talking about `binPack.indentCallSiteOnce`? If so I tried this already and it had no effect,

Before

```scala
    def inteceptBehaviour[T](behavior: Behavior[T], ctx: TypedActorContext[T]): Behavior[T] =
      Behavior
        .start(Behaviors.intercept { () =>
          this.asInstanceOf[BehaviorInterceptor[Any, T]]
        }(behavior), ctx.asInstanceOf[TypedActorContext[Any]])
        .unsafeCast[T]
```

and after

```scala
    def inteceptBehaviour[T](behavior: Behavior[T], ctx: TypedActorContext[T]): Behavior[T] =
      Behavior
        .start(Behaviors.intercept { () =>
            this.asInstanceOf[BehaviorInterceptor[Any, T]]
          }(behavior), ctx.asInstanceOf[TypedActorContext[Any]])
        .unsafeCast[T]
```

When replacing `newlines.implicitParamListModifierPrefer = before` with

```
binPack.unsafeCallSite                   = Always
binPack.unsafeDefnSite                   = Always
binPack.indentCallSiteOnce               = true
```

Reply:
before was broken. after, in this case, has one level of indentation. the indent-once parameter should have an effect where multiple levels were added.

Reply:
Just to be clear, when I said previously there is no difference (for this snippet of code), I meant that 

```
binPack.unsafeCallSite                   = Always
binPack.unsafeDefnSite                   = Always
```

formats the code in the exact same way as

```
binPack.unsafeCallSite                   = Always
binPack.unsafeDefnSite                   = Always
binPack.indentCallSiteOnce               = true
```

Reply:
what about other parts of code? also no difference?

Reply:
There is a difference, so `binPack.indentCallSiteOnce  = true` is definitely doing something in other places, i.e.

Before
```scala
  private[akka] val team =
    new ConcurrentSkipListSet[ActorCell](Helpers.identityHashComparator(new Comparator[ActorCell] {
      def compare(l: ActorCell, r: ActorCell) = l.self.path.compareTo(r.self.path)
    }))
```
After
```scala
  private[akka] val team =
    new ConcurrentSkipListSet[ActorCell](Helpers.identityHashComparator(new Comparator[ActorCell] {
    def compare(l: ActorCell, r: ActorCell) = l.self.path.compareTo(r.self.path)
  }))
```

And another snippet

Before
```scala
  def :+(node: T): ConsistentHash[T] = {
    val nodeHash = hashFor(node.toString)
    new ConsistentHash(nodes ++ ((1 to virtualNodesFactor).map { r =>
        concatenateNodeHash(nodeHash, r) -> node
      }), virtualNodesFactor)
  }
```
After
```scala
  def :+(node: T): ConsistentHash[T] = {
    val nodeHash = hashFor(node.toString)
    new ConsistentHash(nodes ++ ((1 to virtualNodesFactor).map { r =>
      concatenateNodeHash(nodeHash, r) -> node
    }), virtualNodesFactor)
  }
```

Reply:
This has been resolved with the latest release of scalafmt 3.6.1 and the following options

```
binPack.unsafeCallSite                   = always
binPack.unsafeDefnSite                   = always
binPack.indentCallSiteSingleArg          = false
binPack.indentCallSiteOnce               = true
```

Please see newly created PR at https://github.com/apache/incubator-pekko/pull/2